app/src/main/java/nu/yona/app/api/model/User.java

```diff
@@ -257,12 +247,7 @@ public ContentValues getDbContentValues()
 
 	public String getSslRootCertCN()
 	{
-		return this.sslRootCertCN;
-	}
-
-	public void setSslRootCertCN(String sslRootCertCN)
-	{
-		this.sslRootCertCN = sslRootCertCN;
+		return this.getEmbedded().getYonaDevices().getEmbedded().getCurrentDevice().getSslRootCertCN();
```

Comment:
The part ``this.getEmbedded().getYonaDevices().getEmbedded().getCurrentDevice()`` occurs four times. Just put it in a private method.